pdns/dnsdistdist/m4/pdns_check_re2.m4

```diff
@@ -0,0 +1 @@
+../../../m4/pdns_check_re2.m4
```

Comment:
I am inclined to agree with @Habbie on this (although I understand both viewpoints)